lib/bundler/stub_specification.rb

```diff
@@ -68,6 +68,10 @@ def raw_require_paths
       stub.raw_require_paths
     end
 
+    def to_spec
+      stub.to_spec
```

Comment:
Ah yea, normally we do this:

```ruby
stub = Gem::StubSpecification.gemspec_stub(file, install_path.parent, install_path.parent)
          stub.full_gem_path = Pathname.new(file).dirname.expand_path(root).to_s.untaint
          StubSpecification.from_stub(stub)
```

But on second load, we have `stub` which is not a `Gem::StubSpecification`